JSDemos/Demos/PivotGrid/VirtualScrolling/description.md

```diff
@@ -1 +1,7 @@
-The PivotGrid ships with virtual scrolling support. Virtual scrolling improves the performance when displaying large numbers of&nbsp;PivotGrid cells. The PivotGrid in&nbsp;this demo contains approximately 4000&nbsp;cells. To&nbsp;see virtual scrolling in&nbsp;action, scroll the PivotGrid horizontally.
\ No newline at end of file
+If the PivotGrid is bound to a large dataset, you can enable the virtual scroll mode to load only the cells that are in the viewport. This allows you to optimize data load time and reduces resources consumption. When users scroll the PivotGrid, it loads cells to be displayed in the viewport and removes other cells from memory.
```

Comment:
If te component removes the cell from the DOM, then:
```suggestion
If the PivotGrid is bound to a large dataset, you can enable the virtual scroll mode to load cells of the current viewport only. This allows you to optimize data load time and reduce resources consumption. When cells leave the viewport, the PivotGrid removes them from the DOM.
```

Reply:
> resources consumption

Even though we are talking about multiple resources, the compound noun should have "resource" in singular form.

I also changed the structure a bit. Your version starts with information about how the virtual scrolling works ("load only the cells that are in the viewport"), then give a bit of into about about how it can be helpful ("allows you to optimize data load time and reduce resources consumption"), then goes back to how it works ("When cells leave the viewport, the PivotGrid removes them from the DOM."). I put the two similar pieces together in my version.

```suggestion
If the PivotGrid is bound to a large dataset, you can enable the virtual scrolling mode to optimize data load time and reduce resource consumption. In this mode, the PivotGrid loads only the cells that are in the viewport. When cells leave the viewport, the PivotGrid removes them from the DOM.
```